guillotina/factory/content.py

```diff
@@ -161,8 +160,6 @@ def storage(self):
         """
         create root object if necessary
         """
-        request = make_mocked_request('POST', '/')
-        request._db_write_enabled = True
         tm = self.get_transaction_manager()
         txn = await tm.begin()
```

Comment:
Could we set explicit that is write ?

Reply:
Yup, you can specific on the transaction if it is `read_only` or not now.